lms/static/sass/views/_login-register.scss

```diff
@@ -341,6 +341,7 @@ $sm-btn-linkedin: #0077b5;
         width: 100%;
         height: ($baseline*2);
```

Comment:
I would remove the fixed height so the text can define the box, though you'll need to change the padding to padding-top: 10px; padding-bottom: 10px;